File: aws-trigger-benchmarker/runtime/python-http/handler.py
import json
import time

def hello(event, context):

    start_time = int(time.time() * 1000)
    print(event)

    trace_id = event.get('queryStringParameters', {}).get('traceId')


    body = {
        "message": f"traceId: {trace_id}, triggerStartTime: {start_time}, runtime: python",
        "traceIdHttp": trace_id,

    }

    return {"statusCode": 200, "body": json.dumps(body)}

 

File: aws-trigger-benchmarker/runtime/javascript-http-noexpress/handler.js


exports.hello = async (event) => {
  let startTime = Date.now();
  console.log(event);

  
  const traceId = event.queryStringParameters ? event.queryStringParameters.traceId : null;
  const currentTime = new Date().toISOString();

  return {
    statusCode: 200,
    body: JSON.stringify({
      message: `traceId: ${traceId} , triggerStartTime: ${startTime}, runtime: javascript`,
      traceIdHttp: traceId,
      currentTime: currentTime,
    }),
  };
};

